autoware_launch/launch/e2e_simulator.launch.xml

```diff
@@ -25,7 +25,7 @@
   <arg name="lanelet2_map_file" default="lanelet2_map.osm" description="lanelet2 map file name"/>
   <arg name="pointcloud_map_file" default="pointcloud_map.pcd" description="pointcloud map file name"/>
   <!-- Perception -->
-  <arg name="traffic_light_namespace" default="traffic_light" description="traffic light recognition namespace1"/>
+  <arg name="traffic_light_namespace" default="[traffic_light]" description="traffic light recognition namespace"/>
```

Comment:
Does this line means we can set namespace array for each cameras?

Reply:
Yes. We can set any (and multi) camera namespace for tlr.